docs/install/install.md

```diff
@@ -0,0 +1,458 @@
+---
+myst:
+  html_meta:
+    "description": "Installation notes for Plone Classic UI"
+    "property=og:description": "Installation notes for Plone Classic UI"
+    "property=og:title": "Installation notes for pip and buildout based installation paths"
+    "keywords": "Plone, Classic UI, classic-ui, installation, pip, buildout"
+---
+
+(classic-ui-installation-label)=
+
+# Installation
+
+This sections describes how to install Plone 6 Classic UI.
+Examples include the following.
+
+- pip based install method
+- buildout based install method
+
+(classic-ui-installation-pip-label)=
+
+## Quickstart pip based installation
+
+install notes for linux
+
+requirements:
+
+- python3.10 or greater
+- python venv module
+
+in our example we use python3.12
+
+On debian based systems you can install python with following command
+
+```shell
+sudo apt install python3.12 python3.12-dev python3.12-venv
+```
+
+Select a directory of your choice
+
+```shell
+mkdir -p /opt/plone
+cd /opt/plone
+```
+
+Create a virtual environment
+
+```shell
+python3 -m venv ./venv
+```
+
+Activate the virtual environment
+
+```shell
+source ./venv/bin/activate
+```
+
+Install Plone and a helper package
+
+```shell
+pip install Plone cookiecutter
+```
+
+```shell
+cookiecutter -f --no-input --config-file ./instance.yaml https://github.com/plone/cookiecutter-zope-instance
+```
+
+Deactivate the virtual environment
+
+```shell
+deactivate
+```
+
+minimal content for the `instance.yaml` file
+
+```yaml
+# please change the password to a secrue token!
+default_context:
+  initial_user_name: "admin"
+  initial_user_password: "admin"
+  wsgi_listen: "localhost:8080"
+  debug_mode: false
+  verbose_security: false
+  db_storage: "direct"
+  environment: {
+    "zope_i18n_compile_mo_files": true,
+  }
+```
+
+Start the instance for quick test
+
+```shell
+./venv/bin/runwsgi -v instance/etc/zope.ini
+```
+
+Your instance starts in foreground mode, which is only advisable for troubleshooting or for local demonstration purposes.
+
+Now you can call the url `http://localhost:8080` in your browser and you can add a **Classic UI Plone site**.
+
+Let's have fun with Plone!
+
+
+### Example to start the instance via systemd
```

Comment:
I am not sure why the following three examples are in "Install". Aren't these configuration items for how to manage an instance, not install one?

If so, we have a new home for such stuff in [Manage Plone](https://6.docs.plone.org/manage/index.html). The content currently only has Cookieplone based stuff, and we can add buildout and pip based stuff as needed. Please let me know.